### PR TITLE
[WIP] @gndclouds/community will

### PR DESCRIPTION
**Pull request by gndclouds**
Started to apply community page styles form design channel feedback.

### CODE REVIEW

File: components/core/CodeTerminal.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
+import React, { useState, useEffect } from "react";
+import * as Constants from "~/common/constants";
+import * as System from "~/components/system";
+
+import { css, keyframes } from "@emotion/react";
+
+const blinkCursor = keyframes`
+0% {opacity: 0;}
```
Comment (jimmylee) [inline components/core/CodeTerminal.js:8]: nit: formatting

File: components/core/CodeTerminal.js [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+50% {opacity: 1;}
+100% {opacity: 0;}
+`;
+
+const typewriter = keyframes`
+  0%,100% {width: 0;}
+  20%, 80% {width: 10.2em;}
+
```
Comment (jimmylee) [inline components/core/CodeTerminal.js:16]: extra whitespace

File: components/core/CodeTerminal.js [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+  ::-webkit-scrollbar {
+    display: none;
+  }
+`;
+
+const STYLES_WINDOW_NAV = css`
+  border-bottom: 5px solid red;
+`;
+const STYLES_WINDOW_BODY = css`
```
Comment (jimmylee) [inline components/core/CodeTerminal.js:59]: new line

**Review by jimmylee — CHANGES_REQUESTED** [review]

**Comment by gndclouds** [comment]
thanks for catching those! I only meant to submit the community-w so I hadn't checked that file yet.

File: components/core/CodeTerminal.js [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+import * as System from "~/components/system";
+
+import { css, keyframes } from "@emotion/react";
+
+const blinkCursor = keyframes`
+0% {opacity: 0;}
+50% {opacity: 1;}
+100% {opacity: 0;}
+`;
+
```
Comment (jimmylee) [inline components/core/CodeTerminal.js:12]:
```
const blinkCursor = keyframes`
  0% {opacity: 0;}
  50% {opacity: 1;}
  100% {opacity: 0;}
`;
```

File: components/core/CodeTerminal.js [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+const typewriter = keyframes`
+  0%,100% {width: 0;}
+  20%, 80% {width: 10.2em;}
+`;
```
Comment (jimmylee) [inline components/core/CodeTerminal.js:16]:
```
const typewriter = keyframes`
  0%, 100% {width: 0;}
  20%, 80% {width: 10.2em;}
`;
```

File: components/core/CodeTerminal.js [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+  animation-duration: 10s;
+  animation-timing-function: steps(45, end);
+  animation-iteration-count: infinite;
+`;
+
+const CodeTerminal = () => {
+  return (
+    <div css={STYLES_ROOT}>
+      <div css={STYLES_WINDOW}>
+        <div css={STYLES_WINDOW_NAV}>Cat</div>
```
Comment (jimmylee) [inline components/core/CodeTerminal.js:76]: What does Cat mean? Sounds fun :)

**Review by jimmylee — CHANGES_REQUESTED** [review]
Almost :)

File: components/core/CodeTerminal.js (hunk 6 above)
Comment (gndclouds) [reply components/core/CodeTerminal.js:76]: Ha was testing the component ;-)

**Pull request merged**